packages/next/server/next-dev-server.ts

```diff
@@ -33,6 +33,19 @@ import HotReloader from './hot-reloader'
 import { findPageFile } from './lib/find-page-file'
 import { getNodeOptionsWithoutInspect } from './lib/utils'
 
+// NodeJs Version Validation
+var currentNodeVersion = process.versions.node
```

Comment:
let nodeVersion = parseFloat(process.versions.node);
if (nodeVersion < 10.13) {
  throw new Error(`Your current node version ${nodeVersion} is not supported! The minimum supported version is 10.13 and above!`)
}